en/tutorials-and-examples/cms/database.rst

```diff
@@ -153,80 +153,58 @@ able to connect to the database' section has a green chef hat.
     The file **config/app_local.php** is a local override of the file **config/app.php**
     used to configure your development environment quickly.
 
-Creating our First Model
-========================
+Migrations
+==========
 
-Models are the heart of CakePHP applications. They enable us to read and
-modify our data. They allow us to build relations between our data, validate
-data, and apply application rules. Models provide the foundation necessary to
-create our controller actions and templates.
+The SQL statements to create the tables for this tutorial can also be generated
+using the Migrations Plugin. Migrations provide a platform-independent way to
+run queries so the subtle differences between MySQL, PostgreSQL, SQLite, etc.
+don't become obstacles.
```

Comment:
Bear in mind, all these changes that look like complete replacements are actually new content, and the original was split to the new `articles-model.rst` file.